host/debian/postinst

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# Copyright (C) 2002-2005 Flavio Stanchina
+# Copyright (C) 2005-2006 Aric Cyr
+# Copyright (C) 2007 Mario Limonciello
+# Copyright (C) 2009 Alberto Milone
+
+set -e
+
+NAME=xap-hss-host
+PACKAGE_NAME=$NAME
+DEB_NAME=$(echo $PACKAGE_NAME | sed 's,_,-,')
+CVERSION=`dpkg-query -W -f='${Version}' $DEB_NAME | awk -F "-" '{print $1}' | cut -d\: -f2`
+ARCH=`dpkg --print-architecture`
+
+dkms_configure () {
```

Comment:
Configuring DKMS via `postinst` looks like the old way of doing things. Where did you find this script?

I believe that the newer approach is to use a `debian/dkms` file.  For example, see the debian directory for `iptables-netflow-dkms` [here](http://deb.debian.org/debian/pool/main/i/iptables-netflow/iptables-netflow_2.3-5.debian.tar.xz).   Or maybe `bbswitch` is a better [example](http://deb.debian.org/debian/pool/main/b/bbswitch/bbswitch_0.8-8.debian.tar.xz), since it includes the `dkms` file in the upstream source, like you have here. 

Our own `xapea00x` uses a similar [approach](https://github.com/xaptum/xapea00x/tree/debian/stretch/debian), but it creates two packages so the `dkms` files are prefixed with the package names (e.g., `xapea00x.dkms`).

Reply:
I've packaged this more similarly to `xapea00x` after spending a long time trying to copy `bbswitch`.  I think it makes a lot of sense to use a separate Debian branch like our other packages since HSS isn't a Debian specific project. It's all boilerplate code so lets handle this once we merge these changes.